src/main/groovy/org/ysb33r/gradle/gradletest/TestGenerator.groovy

```diff
@@ -306,6 +321,90 @@ class TestGenerator extends DefaultTask {
      */
     @CompileDynamic
     private void copy(
+            final File targetDir,
```

Comment:
This copy method was broken up in an effort to retain as much of the existing code as possible.  instead of the copy method doing the work, the copy method now takes the search results, and separates them out into two calls to a copysub method, one call for gradle build,s one call for gradle kotlin builds.  with appropriate flags being sent for the language type.